### PR TITLE
fix: swapped accounts with wallet

### PR DESCRIPTION
**Pull request by Harshit-Prasad**
This PR swaps the word account with wallet. Mainly in user-menu, accounts-menu and in connect-a-wallet page.

fixes: #2872

### Type of Change
- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes
account-menu
user-menu
connect-a-wallet

### How has this been tested?

Changes are reflected in the extension without breaking anything.

### CODE REVIEW

**Comment by Rithvik-padma** [comment]
@Harshit-Prasad u need to change the translations for other languages too not only for english

**Comment by rolznz** [comment]
> @Harshit-Prasad u need to change the translations for other languages too not only for english

@Rithvik-padma I don't think this is right - we cannot know what the correct translation is for all languages. There is a script created by @reneaaron in `.github/workflows/translations.yml` which should check keys from other languages to see if any need to be removed, and a script you can run here to remove them: `scripts/remove-outdated-translations.js`

File: .yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (reneaaron) [inline .yarnrc.yml:1]: Do we need this file? Otherwise please remove it...

File: yarn.lock
Comment (reneaaron) [inline yarn.lock:1]: Please undo those changes.

**Review by reneaaron — COMMENTED** [review]
@Harshit-Prasad You just need to update the English translation keys and run the script @rolznz mentioned, which will remove the keys from the translated files. 

This will remove existing translations and allows translators to add the new translations (since they are very different from what we have now, e.g. "Add account" vs "Add wallet").

**Comment by AdarshRawat1** [comment]
> @Harshit-Prasad You just need to update the English translation keys and run the script @rolznz mentioned, which will remove the keys from the translated files.

@reneaaron 
I tried running the scripts and I can't seem to get things right.
image

p.s.-  I even tried modifying the script, trying different arguments nothing seems to work

**Comment by reneaaron** [comment]
That's strange. I just executed the commands in your branch locally and they seemed to work. :thinking: 

Maybe you could try to debug the script by adding console.log statements here:
https://github.com/getAlby/lightning-browser-extension/blob/fix/swapped-account-for-wallet/scripts/remove-outdated-translations.js#L13

Otherwise just skip it and I'll have another look before we merge it.

**Pull request closed without merging**